### DISCUSSION TITLE
appointments disappear in google calendar

### DISCUSSION THREAD

**Discussion by chrisdecfd** [kewisch/gdata-provider#420] [Support]
Hello,
from time to time after creating or modifying an appointment (google calendar) all the appointments in the calendar disappears. To solve the problem, I cancel the calendar and I recreate it and it imports automatically all the appointments and it works again until next time. Am I the only one with this trouble ?
Thanks

**Comment by kewisch** [comment]
I've not heard of this specific issue. Can you enable debug logs as noted in the wiki and check around the time they disappear?
